data/remnant/remnant 1 introduction.txt

```diff
@@ -1996,6 +2002,17 @@ mission "Remnant: Taely Intro Log"
 
 
 
+mission "Remnant: Taely Penguin Log"
+	invisible
+	to offer
+		has "Remnant: Taely Intro Log: failed"
+		has "Remnant: Face to Maw 2: done"
+	on offer
+		log "People" "Taely" `One of Taely's main priorities is improving the maneuverability characteristics of Remnant Ships. To this end she has been working on building a new prototype ship based on the void sprites.`
+		fail
+
+
+
```

Comment:
```suggestion
```

Reply:
Instead of creating a whole new mission for this, it would be more efficient to just use actions inside the branches. The branch for already having met Taely would have just this log entry, and the branch for meeting Taely could set both, and pre-emptively set the Intro Log as being offered and done.

Reply:
I don't think it's more efficient to duplicate the same log text that is made in the intro log mission elsewhere.
If that is more efficient though, should I also be deleting the other log missions you have here and applying their logs at the appropriate locations in mission conversations as well?

Reply:
Sorry, I was referring specifically to the Penguin log entry. The other entry can be triggered by various places, so having a single mission for it is easier. Although that being said, ensuring the penguin entry comes after the first one could be problematic then, so maybe the way you have it is better, unless there's some way to specify which order they are in.